package.json

```diff
@@ -1,53 +1,59 @@
-{
-  "name": "setup-deno",
-  "version": "2.1.0",
-  "private": true,
-  "description": "setup deno action",
-  "main": "src/setup-deno.ts",
-  "scripts": {
-    "build": "tsc",
-    "format": "prettier --write **/*.ts",
-    "format-check": "prettier --check **/*.ts",
-    "release": "ncc build && git add -f dist/",
-    "test": "jest"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/denolib/setup-deno.git"
-  },
-  "keywords": [
-    "actions",
-    "deno",
-    "setup"
-  ],
-  "author": "The Deno Authors",
-  "license": "MIT",
-  "dependencies": {
-    "@actions/core": "^1.2.4",
-    "@actions/io": "^1.0.2",
-    "@actions/exec": "^1.0.4",
-    "@actions/http-client": "1.0.8",
-    "@actions/tool-cache": "^1.6.0",
-    "semver": "^6.3.0",
-    "uuid": "3.3.3"
-  },
-  "devDependencies": {
-    "@types/jest": "^26.0.7",
-    "@types/node": "^12.12.12",
-    "@types/semver": "^6.2.0",
-    "@types/uuid": "^3.4.6",
-    "@zeit/ncc": "^0.22.3",
-    "husky": "^4.2.5",
-    "jest": "^26.2.1",
-    "jest-circus": "^26.2.1",
-    "prettier": "^2.0.5",
-    "ts-jest": "^26.1.4",
-    "typescript": "^3.9.7"
-  },
-  "husky": {
-    "skipCI": true,
-    "hooks": {
-      "pre-commit": "npm run build && npm run format"
-    }
-  }
-}
+{
+  "name": "setup-deno",
+  "version": "2.1.0",
+  "private": true,
+  "description": "setup deno action",
+  "main": "src/setup-deno.ts",
+  "scripts": {
+    "build": "ncc build",
+    "check": "run-p format-check lint test",
```

Comment:
I'm not sure where the `run-p` comes from.
This is from `node_modules/.bin`?

Reply:
See https://www.npmjs.com/package/npm-run-all